src/Java/Images/Actor2D.md

```diff
@@ -1,4 +1,4 @@
-Description
+### Description
 
 Example to demonstrate a actor that draws 2D data.
```

Comment:
Example to demonstrate an actor that draws 2D data.

Reply:
I have corrected.